src/Mvc/src/Microsoft.AspNetCore.Mvc.Formatters.Xml/MvcXmlOptions.cs

```diff
@@ -10,39 +11,13 @@ namespace Microsoft.AspNetCore.Mvc.Formatters.Xml
     /// <summary>
     /// Provides configuration for XML formatters.
     /// </summary>
+    /// <remarks>This class is currently empty.</remarks>
```

Comment:
I'm considering removing this class entirely since it's not clear we'll need options or compatibility switches for XML features anytime soon. The downside is the need to remove `AddXmlOptions(...)` and the `AddXmlDataContractSerializerFormatters(...)` and `AddXmlSerializerFormatters(...)` overloads accepting an `Action<MvcXmlOptions>` parameter. **Thoughts?**

Reply:
I think the best thing to to do is just to leave it. It's not costing us anything tangible.

Reply:
Works for me 😺